doc/classes/@GDScript.xml

```diff
@@ -475,6 +475,25 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="is_close">
+			<return type="bool">
+			</return>
+			<argument index="0" name="a" type="float">
+			</argument>
+			<argument index="1" name="b" type="float">
+			</argument>
+			<argument index="2" name="tolerance" type="float" default="1e-05">
+			</argument>
+			<description>
+				Correctly compares two float numbers(third parameter is comparsion precision). Use this function instead == operator on floats.
+				[codeblock]
+				var a = 1.01
```

Comment:
I'm not sure the example is good here, since the fact that the Vector2 has "bad" value is not normal and might change if fixed.
I would just use directly two raw values.

Reply:
The problem is if you compare two raw float values(at least at the same place) they are equal actually(true), so there is no visual difference between == operator and is_close builtin

Reply:
Maybe we could find a basic calculus that allow creating two slightly different values ?

Reply:
Check now, I rewrited it

Reply:
Ok it looks good to me :)